plugins/rhs_plugin/index.js

```diff
@@ -10,14 +10,16 @@ import RHSPlugin from './rhs_plugin.jsx';
 
 function mapStateToProps(state) {
     const rhsPlugins = state.plugins.components.RightHandSidebarComponent;
-    const pluginId = getPluginId(state);
+    const pluginComponentId = getPluginId(state);
```

Comment:
The `getPluginId` selector (and associated actions / reducers) remain confusing, since they imply a plugin id, but it's really this `pluggableId`. Worth fixing that for consistency?

Separately, I wonder if we can just call /this/ variable `pluggableId`, and then dispense with the redefinition of `plugableId` below and simply set `showPluggable` to `Boolean(pluginComponent)`?